core/SConscript.bootloader

```diff
@@ -36,18 +36,24 @@ if TREZOR_MODEL in ('R', ):
     FONT_BOLD='Font_PixelOperator_Bold_8'
     FONT_MONO='Font_PixelOperator_Regular_8'
     FONT_BIG=None
+    FONT_NORMAL_UPPER=None
+    FONT_BOLD_UPPER=None
 elif TREZOR_MODEL in ('T', 'DISC1', 'DISC2'):
     FONT_NORMAL='Font_TTHoves_Regular_21'
     FONT_DEMIBOLD='Font_TTHoves_Regular_21'
-    FONT_BOLD='Font_TTHoves_Bold_17'
+    FONT_BOLD=None
```

Comment:
IIRC if such font is accidentally used it is not caught at compile time but crashes at first rendering. I wonder if changing this to `FONT_BOLD='Font_TTHoves_Bold_upper_17'` could save us some future bootloader headaches.

Reply:
Can't we just use `FONT_BOLD_UPPER` in the bootloader? I'm not sure I follow.

Reply:
are we now perhaps ready to get rid of `FONT_BOLD` etc. constants and redefine them to the target symbols in Rust?

Reply:
> we might be, unless there's be a conflict with #3695

i don't think there would be -- and if it is, we'll resolve it

@obrusvit can you give it a try?

Reply:
@matejcik yes
I will modify the generator further to reuse the same glyphs even in TS3 and then take a look at the constants.

Reply:
The generator is modified as described in a thread above.

As for this:
> are we now perhaps ready to get rid of FONT_BOLD etc. constants and redefine them to the target symbols in Rust? 

We agreed with matejcik that this is out of scope of this PR.

I would like to rebase this on `main` and do a 2nd round of review - necessary because of conflicts in translated strings.